README.md

```diff
@@ -45,57 +41,63 @@ The JFrog VS Code Extension adds JFrog Xray scanning of project dependencies to
 
 The extension also applies [JFrog File Spec JSON schema](https://raw.githubusercontent.com/jfrog/jfrog-cli/master/schema/filespec-schema.json) on the following file patterns: `**/filespecs/*.json`, `*filespec*.json` and `*.filespec`. Read more about JFrog File specs [here](https://www.jfrog.com/confluence/display/JFROG/FileSpec).
 
-<a name="need-a-free-jfrog-instance"></a>
-### Need a FREE JFrog instance?
-Don't have a running JFrog instance? No problem! [Get a FREE instance in the cloud](https://jfrog.com/start-free/).
+## Getting Started
+1. Install the **JFrog** extension in VS Code
+2. Need a FREE JFrog environment in the cloud? [Create one now and connect VS Code to it](#set-up-a-free-jfrog-environment-in-the-cloud)
+3. Already have a working JFrog environment? [Connect VS Code to it](#connecting-vs-code-to-your-jfrog-environment)
+4. [Start](#using-the-extension) using the extension
 
-### Component Tree Icons
-The icon demonstrates the top severity issue of a selected component and its transitive dependencies. The following table describes the severities from lowest to highest:
+## Set Up a FREE JFrog Environment in the Cloud
+Need a FREE JFrog environment in the cloud, so that VS Code can connect to it? Just run one of the following commands in your terminal. The commands will do the following.
```

Comment:
```suggestion
Need a FREE JFrog environment in the cloud, so that VS Code can connect to it? Just run one of the following commands in your terminal. The commands will do the following:
```